Havit.Blazor.Documentation/Pages/Premium/GatewayToPremium.razor.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.JSInterop;
+
+namespace Havit.Blazor.Documentation.Pages.Premium;
+
+public partial class GatewayToPremium
+{
+	[SupplyParameterFromQuery] public string Url { get; set; }
+
+	[Inject] private NavigationManager NavigationManager { get; set; }
+	[Inject] private IJSRuntime JSRuntime { get; set; }
+
+	private IJSObjectReference _jsModule;
+
+	private bool _skipGatewayPage = false;
+	private string _gitHubFolderUrl = "https://github.com/havit/Havit.Blazor.Premium";
+
+	protected override void OnParametersSet()
+	{
+		_gitHubFolderUrl = Uri.UnescapeDataString(Url);
```

Comment:
Query-string parameters get decoded automatically. This might cause double-decoding and corrupt the URL.